src/runtime/entitlements-manager.js

```diff
@@ -595,13 +596,18 @@ export class EntitlementsManager {
 
     const params = new EventParams();
     params.setIsUserRegistered(true);
-    this.deps_
-      .eventManager()
-      .logSwgEvent(
-        AnalyticsEvent.EVENT_UNLOCKED_BY_SUBSCRIPTION,
-        false,
-        params
-      );
+
+    // Avoid logging events from crawlers.
+    if (entitlement.source !== PRIVILEGED_SOURCE) {
```

Comment:
Not sure about this.  Knowing that we used privilege to unlock is useful to publishers who are trying to reconcile their page reads fro  other analytics with subscription unlocks.  Maybe better to filter these out in our logs rather than stop logging them entirely.